config/openiotsdk/lwip/user_lwipopts.h

```diff
@@ -19,12 +19,35 @@
 #ifndef USER_LWIPOPTS_H
 #define USER_LWIPOPTS_H
 
+/**
+ * MEM_LIBC_MALLOC: use malloc/free/realloc provided by your C-library
+ * instead of the lwip internal allocator.
+ */
+#define MEM_LIBC_MALLOC (1)
+
+/**
+ * MEMP_NUM_NETBUF: the number of struct netbufs.
+ */
+#define MEMP_NUM_NETBUF 16
+
+/**
+ * PBUF_POOL_SIZE: the number of buffers in the pbuf pool.
+ *
+ * Some unit tests required more pbuf buffers.
+ */
+#ifdef CHIP_LIB_TESTS
+#define PBUF_POOL_SIZE (1001)
```

Comment:
Can that many actually be allocated in practice on devices?

Reply:
This is a value that allows us to pass some of unit-tests (from AppTests as I remember correctly). Yes, it works for OIS but we need to revisit this in the future and improve the LwIP settings.